src/pages/en/guides/data-fetching.md

```diff
@@ -71,9 +71,13 @@ const weather = json.data
 <h2>{weather.getCityByName.name}, {weather.getCityByName.country}</h2>
 <p>Weather: {weather.getCityByName.weather.summary.description}</p>
 ```
-> 💡 Remember, all data in Astro components is fetched when a component is rendered.
+
+:::note
+Remember, all data in Astro components is fetched when a component is rendered.
 
 Your deployed Astro site will fetch data **once, at build time**. In dev, you will see data fetches on component refreshes. If you need to re-fetch data multiple times client-side, use a [framework component](/en/core-concepts/framework-components/) or a [client-side script](/en/core-concepts/astro-components/#client-side-scripts) in an Astro component.
+:::
+
```

Comment:
```suggestion
:::note
Remember, all data in Astro components is fetched when a component is rendered.
:::

Your deployed Astro site will fetch data **once, at build time**. In dev, you will see data fetches on component refreshes. 

If you need to re-fetch data multiple times client-side, use a [framework component](/en/core-concepts/framework-components/) or a [client-side script](/en/core-concepts/astro-components/#client-side-scripts) in an Astro component.

```

Reply:
You guess right 😅

Reply:
I’ll leave it as is for now. If you really dislike it, we can come back and rework this into some kind of section with this information in.